pkg/utils/utils.go

```diff
@@ -42,15 +42,22 @@ func IsStateApplicable(kubeClient k8sclient.Interface, state *v1.NodeNetworkStat
 }
 
 // GetHostName return the host name from the input
-// if not set, it tries to read from an k8s based on
-// env variable holding the pod's name, and if not possible either
-// tries to read it from the OS
+// if not set, it tries to read from an k8s with the following options
+//  1.- Pods env var POD_NODE_NAME
```

Comment:
This will be enough. we don't need fallback, let's make it simple. I don't think that the env var assignment would ever fail, and if it does, let's fail hard.

Reply:
So we just make the env var mandatory, ok will change